Sources/SmileID/Classes/RectangleDetector/TextDetector.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import Vision
+import CoreImage
+
+
+class TextDetector {
+    let sequenceHandler = VNSequenceRequestHandler()
+    weak var model: DocumentCaptureViewModel?
```

Comment:
We should not couple a TextDetector to a specific ViewModel, there should be a layer of abstraction here. Something like a TextDetectorDelegate, with methods for onTextDetected and onNoTextDetected